usage.txt

```diff
@@ -33,7 +33,7 @@ sign               should contain the identity to be used when running `codesign
 sign-entitlements  the path to entitlements used in signing (mas platform only)
 strict-ssl         whether SSL certificates are required to be valid when downloading Electron.
                    It defaults to true, use --strict-ssl=false to disable checks.
-tmpdir             temp directory. Defaults to system temp directory.
+tmpdir             temp directory. Defaults to system temp directory. Set to `false` to write to `out` directly without temp.
```

Comment:
Well, I was trying to avoid having it say some variant of "temp directory" three times in three sentences - I guess that didn't work out. (I am open to suggestions.)